packages/extension-ui/src/Popup/Signing/Request/SignArea.tsx

```diff
@@ -32,21 +32,23 @@ function SignArea ({ buttonText, className, error, isExternal, isFirst, setError
   const { t } = useTranslation();
 
   useEffect(() => {
-    const lockSigner = async () => {
+    const setSignerLockStatus = async () => {
```

Comment:
I think we can revert this file as well :)

Reply:
@TarikGul I've reverted `SignArea` to use `setTimeout` as it was prior to #1367. I probably should have done this in the first place but I had already done the work of fixing the code before I realized the alarms weren't needed.